src/schema/schema.go

```diff
@@ -27,6 +30,14 @@ func ValidateSchemas(projectState *state.State) error {
 		if err := ValidateConfig(component, config.Local); err != nil {
 			errors.AppendWithPrefix(fmt.Sprintf("config \"%s\" doesn't match schema", projectState.Naming().ConfigFilePath(config.RelativePath())), err)
 		}
+
+		if group, ok := encryption.GetGroupByObject(unencryptedGroups, config); ok {
```

Comment:
Mam k tomu vyhradu, ze kontrola, ci su hodnoty zasifrovane nema nic spolocne s validaciou schemy ... 
teraz ti zavisi `schema` -> `encryption` co su ale 2 rozne veci.

IMHO by si mal:
- v `cli/common.go` premenovat `ValidateSchemas` na `Validate`
- tento kod, ktory komentujem zabalit do funckie `ValidateAllEncrypted` (alebo podobne) v `encryption` package
- z `cli/common.go` `Validate` volat `schema.ValidateSchemas` a `encrytion.ValidateAllEncrypted`
- ak by bol s niecim problem, tak mozeme zavolat

Reply:
jasne, ja som mal v hlave ze to bude uprava v `ValidateSchemas` a uz som nadtym dalej nerozmyslal. Teraz vidim ze to je blbost, upravim :)